server/database/queries/student/getSpecificAlumnit.js

```diff
@@ -0,0 +1,6 @@
+const connection = require('../../config/connection');
+
+const getSpecificAlumnit = (id) =>
+  connection.query('select * from cohort where id = $1', [id]);
```

Comment:
This query return cohort itself not its students as issue want !!
```suggestion
  connection.query('select * from student where cohort_id = $1', [id]);
```